lib/multi-sse.js

```diff
@@ -0,0 +1,128 @@
+var _ = require('lodash');
+var mongojs = require('mongojs');
+var inflect = require('i')();
+var tinySSE = require('tiny-sse');
+var SSE = require('./sse');
+var Promise = require('bluebird');
+var hl = require('highland');
+var JSONAPI_Error = require('./jsonapi-error');
+
+var MultiSSE = function() {
+};
+
+MultiSSE.prototype.init = function(harvesterApp) {
+    this.options = harvesterApp.options;
+    this.db = mongojs(this.options.oplogConnectionString);
+    this.coll = this.db.collection('oplog.rs');
+    this.harvesterApp = harvesterApp;
+    harvesterApp.router.get(this.options.baseUrl + '/changes/stream', this.checkValidRequestMiddleware.bind(this), tinySSE.head(), tinySSE.ticker({seconds: 3}), this.handler.bind(this));
+};
+
+MultiSSE.prototype.checkValidRequestMiddleware = function (req, res, next) {
```

Comment:
seems like a request validation middleware, which is nice :). Might be more obvious if you called it requestValidationMiddleware - do you agree that this change increases readability?